### PR TITLE
Remove use of campus on the property model

### PR DESCRIPTION
**Pull request by nllong**
#### Any background context you want to provide?
The `campus` field has never been used and treated as a second class citizen. The original intent of this field was to denote that a property was a campus, meaning it had a PM Parent Property ID. The campus field was then to be denoted by the PM Parent Property ID equality the PM Property ID.

#### What's this PR do?

* removes the use of Property.campus
* Migration added for removing the field from the columns table. I confirmed that there was only Property.campus fields in the production database.

#### How should this be manually tested?

* @axelstudios  - can you confirm that `campus` was not used in the hashing of the records?
* Monkey test after running the migration.

#### What are the relevant tickets?
#3261

### CODE REVIEW

File: seed/migrations/0180_remove_property_campus.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.2.15 on 2022-11-13 19:45
+
+from django.db import migrations
+
+def remove_old_campus_column(apps, schema_editor):
```
Comment (nllong) [inline seed/migrations/0180_remove_property_campus.py:5]: special part of the migration to remove the `campus` column across all orgs

File: seed/models/columns.py [hunk 2]
```diff
@@ -139,7 +137,6 @@ class Column(models.Model):
 
     # These are columns that should not be offered as suggestions during mapping
     UNMAPPABLE_PROPERTY_FIELDS = [
-        'campus',
```
Comment (nllong) [inline seed/models/columns.py:142]: campus is now mappable!

**Comment by nllong** [comment]
I'm pretty sure that this will require us to rehash the property states. If so, then let's add `state` and `zip` to the normalized address, then rehash then, since it will be required. @axelstudios  -- can you confirm?

File: seed/migrations/0174_fix_ghg_columns.py [hunk 3]
```diff
@@ -3,14 +3,15 @@
 from django.db import migrations
 # from django.core import serializers
 
+
 def remove_old_ghg_columns(apps, schema_editor):
     """create a default BuildingSync column mapping preset for each organization"""
     Organization = apps.get_model("orgs", "Organization")
 
     better_columns = [
-        'better_ghg_reductions_combined',
```
Comment (nllong) [inline seed/migrations/0174_fix_ghg_columns.py:11]: simply formatting

File: seed/migrations/0165_column_column_description.py [hunk 4]
```diff
@@ -18,6 +19,7 @@ def forwards(apps, schema_editor):
                     col.column_description = col.display_name
                 col.save()
 
+
```
Comment (nllong) [inline seed/migrations/0165_column_column_description.py:22]: simply formatting

File: seed/migrations/0169_auto_20220616_1028.py [hunk 5]
```diff
@@ -19,7 +19,7 @@ def add_new_bsync_mappings(apps, schema_editor):
                 'from_field_value': 'text',
                 'from_units': None,
                 'to_table_name': 'PropertyState',
-                'to_field': 'Audit Template Building Id', 
+                'to_field': 'Audit Template Building Id',
```
Comment (nllong) [inline seed/migrations/0169_auto_20220616_1028.py:22]: simply formatting

File: seed/migrations/0170_column_derived_column.py [hunk 6]
```diff
@@ -3,13 +3,14 @@
...
 
+
 def forwards(apps, schema_editor):
     DerivedColumn = apps.get_model("seed", "DerivedColumn")
     Column = apps.get_model("seed", "Column")
 
     with transaction.atomic():
         Column.objects.all().update(derived_column=None)
-        
+
```
Comment (nllong) [inline seed/migrations/0170_column_derived_column.py:13]: simply formatting

File: seed/migrations/0172_update_at_building_id_target.py [hunk 7]
```diff
@@ -13,6 +13,7 @@ def update_at_building_id(apps, schema_editor):
                     mapping['to_field'] = "audit_template_building_id"
             profile.save(update_fields=['mappings'])
 
+
```
Comment (nllong) [inline seed/migrations/0172_update_at_building_id_target.py:16]: simply formatting

**Comment by nllong** [comment]
> I'm pretty sure that this will require us to rehash the property states. If so, then let's add `state` and `zip` to the normalized address, then rehash then, since it will be required. @axelstudios -- can you confirm?

Okay, after inspecting more, I think [this line](https://github.com/SEED-platform/seed/blob/develop/seed/models/properties.py#L404-L405) shows that only the state object is passed to the hash_object method and since the `campus` is on `property` (not `property_state`), then this update will *not* require a database rehashing.

**Review by axelstudios — APPROVED** [review]

**Comment by axelstudios** [comment]
@nllong Looks good to me - Running migrations against the seedv2 database took a grand total of 12 seconds, and no `hash_object`s were harmed.

**Pull request merged**